setup_environment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+###################################################################
+# The purpose of this script is to download, install
+# and configure all files and programs required to
+# take part in developing ActivitySync (Android only).
+###################################################################
+
+CURRENT_USER=$USER
+
+# update package lists from repositories
+sudo apt-get update
+sudo apt-get install git
+
+sudo mkdir /usr/lib/jvm
```

Comment:
Ok, I have no clue how to check versions of java installed on user's machine.
1. `java -version` returns version of jdk set in PATH - gives me nothing, e.g.:
```
>java -version
java version "1.8.0_102"
Java(TM) SE Runtime Environment (build 1.8.0_102-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.102-b14, mixed mode)
```
I still know nothing about users jdk7 and vice versa in case it returns jdk7.

2. Using grep to find out wchich versions of jdk user has is also very poor idea because directory names of java may differ, e.g.:
Installing java from WebUpd8 PPA creates directory */usr/lib/jvm/java-8-oracle*.
On the other hand installing java in the way it's done in this script creates directory */usr/lib/jvm/jdk{version}*

3. Using `update-alternatives --config java` is as bad idea as greping because it uses directories name for identification.

And at this point I run out of ideas.

This is why I put *clean Ubuntu installation* requirement in my guide. I'm not able to write robust script not knowing user's environment, especially when he is not using package manager.

All that comes to my mind is simple prompt asking for users permission to install and configure given java version.